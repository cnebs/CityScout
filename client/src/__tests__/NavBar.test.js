import { shallow } from 'enzyme';
import React from 'react';
import NavBar from '../components/Navbar';
import {
  AppBar,
  Toolbar,
  Typography,
  // InputBase,
  Button
} from '@material-ui/core';

describe('Nav Bar component', () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<NavBar />);
  });
  test('it renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('it renders the corect child components', () => {
    expect(wrapper.find(AppBar).length).toBeGreaterThan(0);
    expect(wrapper.find(Button).length).toBeGreaterThan(0);
    // expect(wrapper.find(InputBase).length).toBeGreaterThan(0);
    expect(wrapper.find(Toolbar).length).toBeGreaterThan(0);
    expect(wrapper.find(Typography).length).toBeGreaterThan(0);
  });
});
