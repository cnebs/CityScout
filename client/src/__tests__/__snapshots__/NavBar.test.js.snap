// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Bar component it renders without crashing 1`] = `ShallowWrapper {}`;
